packages/cli/src/commands/proxy/connect.ts

```diff
@@ -16,6 +16,10 @@ export default class Connect extends ProfileCommand<typeof Connect> {
     ...tunnelServerFlags,
     ...urlFlags,
     ...ux.table.flags(),
+    'env-id': Flags.string({
+      description: 'specify environment id for this app',
+      required: false,
+    }),
```

Comment:
Not a part of this commit, bit perhaps `--access private` is better than the boolean flag

Reply:
I'll fix it in another PR, but I'm not sure what's the right flag here, so it can be done in another PR. (the command is still hidden)
In practice, it defines "default-access" (since it can be overridden by labels), but I think that `--access` or `--default-access` might be ambiguous.